tests/installation/releasenotes.pm

```diff
@@ -5,8 +5,15 @@ use testapi;
 sub run(){
     my $self=shift;
 
-    assert_and_click 'release-notes-button';    # open release notes window
-    assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    if (check_var('VIDEOMODE', 'text')) {
+        send_key "alt-l";   # open release notes window
+        send_key 'alt-s';   # jump to first tab
+        assert_screen 'release-notes-tab-sle';
+    }
+    else {
+        assert_and_click 'release-notes-button';    # open release notes window
+        assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    }
```

Comment:
Lets's see if that is a bug or feature:
https://bugzilla.suse.com/show_bug.cgi?id=936743